app/services/price_policy_builder.rb

```diff
@@ -14,23 +14,25 @@ def self.get_new_policies_based_on_most_recent(product, start_date)
     new(product, start_date).new_policies_based_on_most_recent
   end
 
-  def self.create_nonbillable_for(product)
-    PricePolicy.create(
-      type: "#{product.type}PricePolicy",
-      product: product,
-      start_date: 1.month.ago,
-      expire_date: 75.years.from_now,
-      price_group: PriceGroup.nonbillable,
-      usage_rate: 0,
-      minimum_cost: 0,
-      cancellation_cost: 0,
-      usage_subsidy: 0,
-      unit_cost: 0,
-      unit_subsidy: 0,
-      can_purchase: true,
-      charge_for: "reservation",
-      note: "Price rule automatically created because of billing mode"
-    )
+  def self.create_skip_review_for(product)
+    PriceGroup.globals.each do |price_group|
```

Comment:
What about non-global price groups?  I think it's possible to add users and payment sources to non-global price groups, but not sure if this breaks things or not.  Please add specs to confirm.

Reply:
Also, we're adding a new dependency between Price Groups and price rules here.  I think `setup_global` should create price rules for all skip review products.

Reply:
Also - can you add a rake task for updating any existing skip review products?

Reply:
Users or accounts should always belong to at least on global price group - so I think we are good here